php/getBudgetTable.php

```diff
@@ -6,28 +6,29 @@
 try {
 	if (get_class($this) == 'Vanderbilt\InstituteBudget\InstituteBudget') {
 		// survey page shim, may eventually want to convert $module to $this everywhere in this file (although that will break including this php script into stand-alone module pages)
+        /** @var \Vanderbilt\InstituteBudget\InstituteBudget $module */
 		$module = $this;
 	}
 	$arms = $module->getArms();
 	$procedures = $module->getProcedures();
-    foreach ($procedures as $proc_i => $procedure) {
-        $proc_name = $procedure->name;
-        $proc_cost = $procedure->cost;
-    }
+    //foreach ($procedures as $proc_i => $procedure) {
```

Comment:
This is commented, is it correct?

Reply:
It's actually not needed at all here, so I've removed it.